src/js/components/organisms/hero-cover/hero-cover.scss

```diff
@@ -0,0 +1,43 @@
+/** @format */
+
+.hero--cover {
+    @extend %reset-last-margin;
+    position: relative;
+    padding-bottom: $buildingUnit * 5;
+
+    &-reset p:last-child {
+        margin-bottom: 0;
+    }
+
+    .copy--intro,
+    .link--large {
+        color: $colorWhite;
+    }
+}
+
+.hero-cover__headline {
+    color: $colorWhite;
+}
+
+.hero-cover__image--full {
+    @include object-fit(cover);
```

Comment:
This is a `<div>`. `object-fit` only works on replaced media such as `<img>`. Also, the mixin is abstracting away useful information.

Reply:
Not going into css yet. Only moving components for now. Css is the next stage / stage.